functional/tests/utilities.mjs

```diff
@@ -223,4 +223,39 @@ async function mfState(on, extraConfig = "") {
   });
 }
 
-export { fireboltCommand, callApi, callMfCli, mfState, killPort };
+/**
+ * @param {String} command the command which we need to execute
+ * @param {Number} port ws port to connect
+ * @param {String} user to pass the userID
+ * @param {Number} eventCommands how many commands will be sent once eventListener is active
+ * @returns Promise yielding the response on resolve()
+ */
+const mfEventListener = (command, port, user) => {
+  return new Promise((resolve) => {
+    const wsClientURL = `${wsClient}${port || 9998}${user ? `/${user}` : ''}`;
+    const ws = new WebSocket(wsClientURL);
+    ws.on("open", () => {
```

Comment:
Do we want to put this after "message"? That way we've properly defined how to receive the response _before_ we send the message across?